tests/precalculations/test_text_preprocessor.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from shift_detector.precalculations.Tokenizer import TokenizeIntoWords
+
+class TestTokenizerFunctions(unittest.TestCase):
+
+    def test_tokenize_into_words(self):
+        normal = "This. is a'n example, ,, 12  35,6  , st/r--ing    \n test."
```

Comment:
Is it desired behavior to ignore all commas in this string?

Reply:
yes